qiita_db/data.py

```diff
@@ -576,40 +583,6 @@ def clear_filepaths(self):
         # self.studies should only have one element, thus self.studies[0]
         move_filepaths_to_upload_folder(self.studies[0], filepaths)
 
-    def remove_filepath(self, fp):
```

Comment:
Yup, it was not used, sorry I forgot to comment that on the PR. I checked and it was not used. This function was intended to remove a single filepath from the raw data, but we are not supporting right now because it gets tricky: we are checking if the raw data has the same number of barcode files and forward reads, so removing a single file will break this checks. Thus, we are allowing just removing all using `clear_filepaths`

Also, this function was using `purge_filepaths` instead of `move_filepaths_to_upload_folder`.

Given that it was not used, and I don't see a use for it soon; and it's a developer burden to maintain it, I think is better to remove it and we will re-add it if we actually need it.